src/Store/KeyBindingsStoreConnector.re

```diff
@@ -145,12 +145,23 @@ let start = () => {
       },
       {
         key: "<CR>",
-        command: "insertBestCompletion",
-        condition: "suggestWidgetVisible" |> parseExp,
+        command: "acceptSelectedSuggestion",
+        condition:
+          "acceptSuggestionOnEnter && suggestWidgetVisible" |> parseExp,
```

Comment:
When / where does the `acceptSuggestionOnEnter` condition get set?

Reply:
It doesn't. Good catch. This was copied from the vscode keybindings, but I've forgotten to implement the configuration setting that would set this.

Reply:
Fixed in https://github.com/onivim/oni2/pull/1170/commits/67b1043f5bd2cbe268452fa2fd8b99c14c13e0b7

Reply:
It's a bit confusing to use the term "suggestion" in keybindings and configuration, and "completion". Is there a reason for using the latter instead?

Reply:
It's an unfortunate consequence of being VSCode-compatible - they actually use `suggest` and `completion` both!

In the extension host: 
```
	$provideCompletionItems(handle: number, resource: UriComponents, position: IPosition, context: modes.CompletionContext, token: CancellationToken): Promise<SuggestResultDto | undefined>;
	$resolveCompletionItem(handle: number, resource: UriComponents, position: IPosition, suggestion: modes.CompletionItem, token: CancellationToken): Promise<modes.CompletionItem>;
	$releaseCompletionItems(handle: number, id: number): void;|
```
https://github.com/onivim/vscode-exthost/blob/a25f426a04fe427beab7465be660f89a794605b5/src/vs/workbench/api/node/extHost.protocol.ts#L971

(And note they even use `suggestion: modes.CompletionItem` above).

But in the keybindings / configuration, they use `suggest`. I figured it was best to keep the 'code-facing' parts across the protocol consistent - and use completion up until the point where keybindings / configuration enter the picture.

Either way - there will be some inconsistency, either from keybindings -> our code, or our code -> extension host.

Reply:
Ah, thanks for explaining. We're using "completions" to talk about the feature generally though, in issues and when talking with users. We should probably use "suggestions" instead and just deal with the confusion ourselves.

Reply:
Good point - it would make sense in the documentation to reference "suggestions" so that the configuration, documentation, and key bindings are consistent.